numpy/__init__.pyi

```diff
@@ -2998,6 +3009,17 @@ class generic(_ArrayOrScalarCommon):
         self: _ScalarType, *shape: SupportsIndex, order: _OrderACF = ...
     ) -> ndarray[Any, dtype[_ScalarType]]: ...
 
+    def bit_count(
+        self,
+        out: None | NDArray[Any] = ...,
+        *,
+        where: _ArrayLikeBool_co = ...,
+        casting: _CastingKind = ...,
+        order: _OrderKACF = ...,
+        dtype: DTypeLike = ...,
+        subok: bool = ...,
+    ) -> NDArray[Any]: ...
```

Comment:
```suggestion
    ) -> Any: ...
```
See https://github.com/numpy/numpy/pull/19355/files#r677587591.

Reply:
Fixed in same commit, thanks.